documentation/examples/prometheus-kubernetes.yml

```diff
@@ -74,19 +68,53 @@ scrape_configs:
     replacement: $1:$2
   - action: labelmap
     regex: __meta_kubernetes_service_label_(.+)
-    replacement: $1
   - source_labels: [__meta_kubernetes_role]
     action: replace
-    regex: (.+)
     target_label: kubernetes_role
-    replacement: $1
   - source_labels: [__meta_kubernetes_service_namespace]
     action: replace
-    regex: (.+)
     target_label: kubernetes_namespace
-    replacement: $1
   - source_labels: [__meta_kubernetes_service_name]
```

Comment:
Ah, generally. The `job` label should be quite exactly what the service name is, so all targets within a job should export the same set of metrics. So when using this example config I always relabeled this into `job` rather than `kubernetes_name`.

Objections to adjust the config accordingly?